angular.json

```diff
@@ -28,8 +28,10 @@
               "src/assets"
             ],
             "styles": [
```

Comment:
@Yuriy-27, please create vendor.scss file in your assets styles and put there any libs scss imports,
and vendor.scss will be imported to main.scss and ONLY main.scss will be imported to angular.json.

Try to do it please